src/goto-instrument/contracts/contracts.cpp

```diff
@@ -1458,10 +1458,10 @@ void code_contractst::apply_loop_contracts(
         continue;
 
       // Store loop number for
-      // ASSIGN ENTERED_LOOP = TRUE
+      // ASSIGN ENTERED_LOOP = true
```

Comment:
Not sure what to make of these two comments, but they seem to say the opposite thing to eachother.